### PR TITLE
added comment

### PR DESCRIPTION
**Pull request by simobgl**
one

### CODE REVIEW

**Review by KaliMik85 — APPROVED** [review]

**Review by Geogufo — APPROVED** [review]

File: cmd/mmApi/main.go [hunk 1]
```diff
@@ -11,6 +11,7 @@ import (
 // =====================================================================================================================
 // MAIN
 func main() {
+	// comment
```
Comment (Geogufo) [inline cmd/mmApi/main.go:14]: Incorrect

**Comment by KaliMik85** [comment]
ok

**Comment by simobgl** [comment]
> 

thanks

**Review by Geogufo — CHANGES_REQUESTED** [review]

File: cmd/mmApi/main.go (hunk 1 above)
Comment (simobgl) [reply cmd/mmApi/main.go]: solved via 119903051680e4b822fd70afab42bcb36005e90e

**Review by Geogufo — APPROVED** [review]

**Review by KaliMik85 — APPROVED** [review]

**Review by KaliMik85 — APPROVED** [review]

**Review by KaliMik85 — APPROVED** [review]

**Review by Geogufo — APPROVED** [review]

**Review by KaliMik85 — APPROVED** [review]

**Review by KaliMik85 — APPROVED** [review]

**Review by Geogufo — APPROVED** [review]

**Comment by KaliMik85** [comment]
ok

**Pull request merged**